common/esxi_get_guest_ids.yml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Get all supported guest ids for a hardware version on ESXi server
+# Parameter:
+#   esxi_hardware_version: The hardware version on ESXi server, e.g. 13, 14, 15, etc.
+# Return:
+#   esxi_guest_ids: A list of all supported guest ids for hardware version
+
+- block:
```

Comment:
I think here `esxi_hardware_version` can be a required parameter must to be set when calling this task, and no need to call `esxi_get_hardware_versions.yml` one more time.

Reply:
Add an assert task for checking esxi_hardware_version